crates/common/Cargo.toml

```diff
@@ -10,10 +10,10 @@ full-serde = []
 [dependencies]
 anyhow = { workspace = true }
 bitvec = { workspace = true }
-fake = { version = "2.5.0", features = ["derive"] }
+fake = { version = "2.8.0", features = ["derive"] }
```

Comment:
Can become a workspace dep.

Reply:
Quite a few other instances, can disregard this comment and leave it for a future PR.

Reply:
I've converted _some_ of our most commonly used dependencies to workspace deps. See the follow-up commits.